lib/vrl/stdlib/benches/benches.rs

```diff
@@ -936,6 +938,15 @@ bench_function! {
     }
 }
 
+bench_function! {
+    keys => vrl_stdlib::Keys;
+
+    literal {
+        args: func_args![value: value!({"key1": "val1", "key2": "val2"})],
+        want: Ok(["key1", "key2"]),
```

Comment:
if you use the `value!` macro, it can correctly create a `Value::Array` for you, such as
`value!(["key1", "key2"])`

Reply:
fixed this in most recent commit